packages/eui/src/components/tabs/tabs.tsx

```diff
@@ -67,9 +75,36 @@ export const EuiTabs = forwardRef<EuiTabRef, EuiTabsProps>(
       bottomBorder && styles.bottomBorder,
     ];
 
+    const handleKeyDown = (event: KeyboardEvent) => {
+      const currentEl = elementRef.current;
+
+      const tabs = currentEl?.querySelectorAll('[role="tab"]');
+      if (!tabs) return;
+
+      const currentIndex = Array.from(tabs).findIndex((tab) =>
+        tab.matches(':focus')
+      );
+
+      if (event.key === keys.ARROW_LEFT) {
+        const previousIndex = (currentIndex - 1 + tabs.length) % tabs.length;
```

Comment:
I've never seen this shortcut math for looping back around to the first/last item in the array before 🤯 Can we add a quick inline code comment here explaining what's happening? e.g.,

```suggestion
        const previousIndex = (currentIndex - 1 + tabs.length) % tabs.length; // % loops back to the last item in the array
```

\+ same for the `nextIndex` `%` logic

Reply:
@cee-chen A neat math trick ✨ but you're right, it's not too clear. We could just do:
```js
const previousIndex = (currentIndex === 0 ? tabs.length : currentIndex) - 1;
```
What do you think?

Reply:
I'm a fan of the ternaries - the new code you suggested feels explicit and easy to understand to me. Let's go with that! ✨